pkg/controller/registry/registry_client.go

```diff
@@ -0,0 +1,112 @@
+//go:generate go run github.com/maxbrunsfeld/counterfeiter/v6 -o resolver/fakes/fake_registry_client_interface.go . RegistryClientInterface
+package registry
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	registryapi "github.com/operator-framework/operator-registry/pkg/api"
+	"github.com/operator-framework/operator-registry/pkg/client"
+	opregistry "github.com/operator-framework/operator-registry/pkg/registry"
+)
+
+type ChannelEntryStream interface {
+	Recv() (*registryapi.ChannelEntry, error)
+}
+
+type RegistryClientInterface interface {
```

Comment:
nit: exported name stutters `registry.RegistryClientInterface` (as opposed to `registry.ClientInterface`)

Reply:
ack